spec/factories/rooms.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: rooms
+#
+#  id            :integer          not null, primary key
+#  number        :integer          not null
+#  conference_id :integer
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :room do
+    number { Faker::Number.between(1, 25) }
```

Comment:
When we put in validations for rooms to have different numbers this might be a problem.
For now let's not worry about that!